week2_PS_draft.ipynb

```diff
@@ -0,0 +1,523 @@
+{
```

Comment:
<p>change line 2 to: mo_precip_1971_2019 = np.loadtxt(fname= "data/mo-precip-1971-2019-w-avg.csv", delimiter = ",")</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/CUB-Computational-Tools/2020-ps-MzCorona/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>